sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/CloseReason.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Processor
+{
+    /// <summary>
+    ///   The reason for stopping event processing for a given partition.
+    /// </summary>
+    ///
+    public enum CloseReason
```

Comment:
The skeleton renames this to `ProcessingStoppedReason`,  though I'm perfectly fine if this flows in at some point in the future.   Just noting for awareness.

[reference](https://gist.github.com/jsquire/3fadc1ea57594c31f1c39366c0d7b530#azuremessagingeventhubsprocessor)

Reply:
I also wonder if this shouldn't be a nested type inside of the `PartitionProcessingStoppedContext` class; do we actually use it in a scope outside of that?

Reply:
Renaming will happen in the next PR. We could move it to the `PartitionProcessingStoppedContext` class, but the full name would get too big.